src/pretix/plugins/socialauth/schemas/oauth2_params.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel, Field
+
+
+class OAuth2Params(BaseModel):
+    response_type: str = Field(default="code")
```

Comment:
To make code clean, please don't use `Field` when you just need to define default value. This is enough:

```py
response_type: str = 'code'
```